map/apply-scheduled-updates-to-preplanned-map-area/gradlew.bat

```diff
@@ -5,7 +5,7 @@
 @rem you may not use this file except in compliance with the License.
 @rem You may obtain a copy of the License at
 @rem
-@rem      https://www.apache.org/licenses/LICENSE-2.0
+@rem      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
not sure if we should be changing gradlew.bat files: but is this just automatic from having made the above change?

Reply:
Yeah this will have crept in with a find & replace. We can undo this one just to be safe

```suggestion
@rem      https://www.apache.org/licenses/LICENSE-2.0
```